pontoon/tags/tests/conftest.py

```diff
@@ -0,0 +1 @@
+from fixtures import *  # noqa
```

Comment:
Make this an explicit relative import via `.fixtures`? To prevent that from happening, maybe also a `from __future__ import absolute_imports` ?